vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -536,27 +410,32 @@ above would look like this:
         }
       }
     }
-
-## Widgets
-
+ 
+### Widgets
+ 
 Different types of widgets can be applied by passing the type via
-"ui:widget". Property tells the form which UI widget should be used to
-render a field. You can read about react-jsonschema-form supported
-widgets here.
-
-## Building From source
-
-Reference the component projects for instructions to build from source.
-<https://github.com/entando/entando-process-driven-plugin>
-<https://github.com/entando/entando-process-driven-plugin/tree/master/widgets>
-<https://github.com/entando/pda-redhatpam-engine>
-<https://github.com/entando/pda-core-engine>
-
-## Code Style
-
-See: <https://github.com/entando/entando-code-style>
-
-## Sonar
-
-<https://sonarcloud.io/organizations/entando/projects>
-
+`ui:widget`. This property specifies the widget to use when the form
+renders a UI field. See the documentation to learn about widgets supported by the `react-jsonschema-form`.
+ 
+### Building from Source
+ 
+Reference the component projects for instructions to build from source code:
+
+- <https://github.com/entando/entando-process-driven-plugin>
+- <https://github.com/entando/entando-process-driven-plugin/tree/master/widgets>
+- <https://github.com/entando/pda-redhatpam-engine>
+- <https://github.com/entando/pda-core-engine>
+ 
+### Code Style
+ 
+Refer to: <https://github.com/entando/entando-code-style>
+ 
+### Sonar
+ 
+Refer to: <https://sonarcloud.io/organizations/entando/projects>
+
+## Resources
+
+### Source Code
+
+The source code for the Entando PDA plugin can be found on GitHub [here](https://github.com/entando-samples/entando-process-driven-plugin-bundle), along with our other open source examples and tutorials.
```

Comment:
The content under Building from Source can be moved here. The current link here is pointing to the bundle rather than the source code it's constructed from.